### PR TITLE
Add Debugging Partial Path Queries for Python

### PR DESCRIPTION
**Pull request by GeekMasher**

### CODE REVIEW

File: python/debugging/PartialPathsFromSource.ql [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+import semmle.python.dataflow.new.BarrierGuards
+import DataFlow::PartialPathGraph
+import semmle.python.ApiGraphs
+// Helpers
+import github.Helpers
+import github.LocalSources
+// Bring in the CommandInjection config
+private import semmle.python.security.dataflow.CommandInjectionCustomizations
+
+// Parcial Graph
```
Comment (aegilops) [inline python/debugging/PartialPathsFromSource.ql:27]: Can you fix the typo please? Partial, not parcial

**Review by aegilops — CHANGES_REQUESTED** [review]
Looks fine to me apart from that one typo

File: python/debugging/PartialPathsFromSource.ql (hunk 1 above)
Comment (GeekMasher) [reply python/debugging/PartialPathsFromSource.ql]: Fixed typos

**Comment by NickLiffen** [comment]
@aegilops as you left feedback, would love some 👀 on this 🙇

**Review by aegilops — APPROVED** [review]
Thanks for the minor changes.

I confess I haven't run this myself, but I've seen the results you showed me.

**Pull request merged**